### PR TITLE
[Improve][Docs][setup] Add Building sub module

### PR DESCRIPTION
**Pull request by liugddx**
## Purpose of this pull request
 Add Building sub module

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/en/contribution/setup.md [hunk 1]
```diff
@@ -35,12 +35,24 @@ Otherwise, your code could not start in JetBrains IntelliJ IDEA correctly.
...
 ```
 
+### Building sub module
+
+If you want to build submodules separately,you can use the follow command to compile and package.
+
+```ssh
+# This is an example of building the redis connector separately
+
+ mvn clean package -pl seatunnel-connectors-v2/connector-redis -am -DskipTest
```
Comment (ashulin) [inline docs/en/contribution/setup.md]:
```suggestion
 mvn clean package -pl seatunnel-connectors-v2/connector-redis -am -DskipTest -T1C
```

Comment (TyrantLucifer) [reply docs/en/contribution/setup.md]: +1

**Review by EricJoy2048 — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Comment by CalvinKirs** [comment]
This is one way, but not the most ideal, in fact, https://github.com/apache/incubator-seatunnel/pull/1980
had solved the problem, but now it does not work properly

**Pull request merged**